### PR TITLE
Pull in rollup to correctly bundle for different platforms

### PR DESCRIPTION
**Pull request by henrymgarrett**

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -13,7 +12,21 @@ jobs:
       - name: Check out source code
         uses: actions/checkout@v2
 
+      - name: Setup Node
```
Comment (henrymgarrett) [inline .github/workflows/npm-publish.yml:15]: Changes here

File: package.json [hunk 2]
```diff
@@ -1,9 +1,20 @@
 {
   "name": "@jupiterone/insights-dashboards",
```
Comment (henrymgarrett) [inline package.json:2]: Changes in here

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import resolve from "@rollup/plugin-node-resolve";
```
Comment (henrymgarrett) [inline rollup.config.js:1]: Main changes are here.  Bundling as a library via rollup

**Review by hallsamuel90 — APPROVED** [review]

**Comment by erkangz** [comment]
@henrymgarrett i just realized this broke the links in README, and the links we've shared before

**Pull request merged**